rtengine/improccoordinator.cc

```diff
@@ -666,12 +714,11 @@ void ImProcCoordinator::updatePreviewImage(int todo, bool panningRelatedChange)
             }
         }
 
-        oprevi = orig_prev;
-
```

Comment:
This doesn't look right. It might have been misplaced after reordering the processes in 7d310e688c9837d31c262678df3c71a278ce11f0, but I'm not sure. Could be related to a bug I found where enabling dehaze makes spot removal ineffective.